tests/foreman/cli/test_domain.py

```diff
@@ -38,6 +39,50 @@ def test_positive_create(self, options):
         except CLIFactoryError as err:
             self.fail(err)
 
+    def test_create_domain_with_location(self):
+        """@Test: Check if domain with location can be created
+
+        @Feature: Domain - Positive create
+
+        @Assert: Domain is created and has new location assigned
+
+        """
+        try:
+            location = make_location()
+            domain = make_domain({
+                'location-ids': location['id']
+            })
+        except CLIFactoryError as err:
+            self.fail(err)
+
+        self.assertIn(
+            location['name'],
+            domain['locations'],
+            "Location wasn't assigned to domain"
+        )
+
+    def test_create_domain_with_organization(self):
+        """@Test: Check if domain with organization can be created
+
+        @Feature: Domain - Positive create
+
+        @Assert: Domain is created and has new organization assigned
+
+        """
+        try:
+            org = make_org()
+            domain = make_domain({
+                'organization-ids': org['id']
+            })
+        except CLIFactoryError as err:
+            self.fail(err)
+
+        self.assertIn(
+            org['name'],
+            domain['organizations'],
+            "Organization wasn't assigned to domain"
```

Comment:
Please drop this error message too. I think it is unnecessary in the same way that `"Location wasn't assigned to domain"` is unnecessary.